src/snippets/article-vor.v1.yaml

```diff
@@ -19,5 +19,8 @@ allOf:
             properties:
                 thumbnail:
                     $ref: ../misc/image.v1.yaml
-            required:
-              - thumbnail
+            anyOf:
```

Comment:
This should not have changed. This was an error. Need to work on this some more.